editor/components/post-sticky/test/index.js

```diff
@@ -9,54 +9,51 @@ import { shallow } from 'enzyme';
 import { PostStickyCheck } from '../check';
 
 describe( 'PostSticky', () => {
-	const user = {
-		data: {
-			post_type_capabilities: {
-				edit_others_posts: true,
-				publish_posts: true,
-			},
-		},
-	};
-
 	it( 'should not render anything if the post type is not "post"', () => {
 		const wrapper = shallow(
-			<PostStickyCheck postType="page" user={ user }>
+			<PostStickyCheck postType="page" post={ {
+				_links: {
+					self: {
+						href: 'https://w.org/wp-json/wp/v2/posts/5',
```

Comment:
Should this be `/page/5` if `postType="page"`? (Doesn't change test result, admittedly)

Reply:
53ce334e